### ISSUE TITLE
About Tab Does Not Display Version 

### ISSUE THREAD

**Issue by Grossdm** [mrdokenny/Cookie-AutoDelete#443]
I noticed that the [about tab](https://mega.nz/#!Gz4nlYqa!5N5OON0oF4CH8rIP00f3VBKrZNgH_FdbvI3tJxAi5tA) did not show the current version of CAD.

This could be helpful to users.

**Comment by kennethtran93** [comment]
Although the 'welcome' tab shows the change log with the most current/recent changes listed first (thus showing the version there), putting it in the about tab is a good call as well.

Probably a good first issue unless Kenny has a quickfix for this.

**Comment by Grossdm** [comment]
Thank you, @kennethtran93!

Before I just guess what this is, please explain the "good first issue" label.

Doug®

**Comment by kennethtran93** [comment]
@Grossdm it's just a way for people who want to contribute to the code but may not know where to begin.

**Comment by Grossdm** [comment]
Ah, that's what I had guessed, but I wanted to be sure.

Thanks again.

**Comment by kennethtran93** [comment]
Would've done this sooner had I got my laptop working again, but there it is.  Screenshot is in PR

**Comment by Grossdm** [comment]
Hmm. How to find that PR...?

**Comment by kennethtran93** [comment]
There's a link above - click on the referenced 474 above to see it.

**Comment by Grossdm** [comment]
Sorry - I'm using an app to access GitHub at the moment. Apparently it is lacking in some ways (such as the missing PR link).

**Comment by Grossdm** [comment]
This is interesting.

I've tried viewing from a browser now, and each time I try to access `https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/pull/443` it redirects to `https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/issues/443`.

I can view other "open" pull requests, though.

Update: I had to add inline code markup to preserve the URLs.

In the pain text, **both** were converted to `#443` and linked to the issue.
